lib/galaxy/datatypes/sequence.py

```diff
@@ -1206,25 +1206,24 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
         >>> Axt().sniff( fname )
         False
         """
-        headers = get_headers(file_prefix, None)
-        if len(headers) < 4:
+        headers = get_headers(file_prefix, None, count=4, comment_designator="#")
```

Comment:
Can't you have an axt file of just 3 lines in theory?

Reply:
haha, I just pushed a fix for just that 🦅👀